configs/vid/fgfa/fgfa_faster_rcnn-resnet101-dc5_8x1bs-7e_imagenetvid.py

```diff
@@ -1,4 +1,4 @@
-_base_ = ['./selsa_troialign_faster_rcnn_r50_dc5_7e_imagenetvid.py']
+_base_ = ['./fgfa_faster_rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py']
```

Comment:
```suggestion
_base_ = ['./fgfa_faster-rcnn-resnet50-dc5_8x1bs-7e_imagenetvid.py']
```

modify it for all config & readme.md & metafile.yml of FGFA and SELSA